packages/server/test/__snapshots__/course.integration.ts.snap

```diff
@@ -15,6 +15,7 @@ Object {
     },
   ],
   "queues": Array [],
+  "timezone": null,
```

Comment:
should we add a tz to the test? i know they're nullable, but in the future, our intention is for them to *not* be right?